src/Microsoft.AspNetCore.Mvc.Core/ContentResult.cs

```diff
@@ -67,13 +67,17 @@ public override Task ExecuteResultAsync(ActionContext context)
 
             if (Content != null)
             {
-                var bufferingFeature = response.HttpContext.Features.Get<IHttpBufferingFeature>();
-                bufferingFeature?.DisableResponseBuffering();
+                var data = resolvedContentTypeEncoding.GetBytes(Content);
```

Comment:
We need to figure out how that works, we can't just do unbounded buffering or unbounded allocation (like this does). 

For now let's just get rid of the call to `DisableResponseBuffering`